appinfo/routes/routesRoomController.php

```diff
@@ -104,5 +104,7 @@
 		['name' => 'Room#setSIPEnabled', 'url' => '/api/{apiVersion}/room/{token}/webinar/sip', 'verb' => 'PUT', 'requirements' => $requirementsWithToken],
 		/** @see \OCA\Talk\Controller\RoomController::setMessageExpiration() */
 		['name' => 'Room#setMessageExpiration', 'url' => '/api/{apiVersion}/room/{token}/message-expiration', 'verb' => 'POST', 'requirements' => $requirements],
+		/** @see \OCA\Talk\Controller\RoomController::setPreHistory() */
+		['name' => 'Room#setPreHistory', 'url' => '/api/{apiVersion}/room/{token}/pre-history', 'verb' => 'POST', 'requirements' => $requirements],
```

Comment:
```suggestion
		['name' => 'Room#setShowHistory', 'url' => '/api/{apiVersion}/room/{token}/pre-history', 'verb' => 'POST', 'requirements' => $requirements],
```

As general wording I would go with "Show history" for the feature, from the settings to API and database field.